docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+</section>
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-diff">
+        <figcaption>diff Syntax Highlighting Example</figcaption>
+        {% code 'diff' %}
+        -        Index: languages/ini.js
+        -        ===================================================================
+        -        --- languages/ini.js    (revision 199)
+        -        +++ languages/ini.js    (revision 200)
+        -        @@ -1,8 +1,7 @@
+        -         hljs.LANGUAGES.ini =
+        -         {
+        -           case_insensitive: true,
+        -        -  defaultMode:
+        -        -  {
+        -        +  defaultMode: {
+        -             contains: ['comment', 'title', 'setting'],
+        -             illegal: '[^\\s]'
+        -           },
+        -
+        -        *** /path/to/original timestamp
+        -        --- /path/to/new      timestamp
+        -        ***************
+        -        *** 1,3 ****
+        -        --- 1,9 ----
+        -        + This is an important
+        -        + notice! It should
+        -        + therefore be located at
+        -        + the beginning of this
+        -        + document!
+        -
+        -        ! compress the size of the
+        -        ! changes.
+        -
+        -          It is important to spell
+        -      {% endcode %}
+    </figure>
+</section>
+
+<section><!-- Syntax Highlighting -->
+
+    <figure data-dom-diff="typography-syntax-html">
+      <figcaption>HTML Syntax Highlighting Example</figcaption>
+      {% code 'html' %}
+        <script type="text/javascript" src="path/to/script.js"></script>
+        <link rel="stylesheet" href="path/to/style.css" />
+        <!-- HTML Comment -->
+        <form method="POST" action="submit.html">
+          <fieldset>
+            <legend>Login</legend>
+            <p>
+              <label for="username">Username</label>
+              <input type="text" id="username" name="username" />
+            </p>
+            <p>
+              <label for="password">Password</label>
+              <input type="password" id="password" name="password" />
+            </p>
+          </fieldset>
+          <p>
+            <button type="submit">Log In</button>
+          </p>
+          <p>
+            <img src="path/to/file.png" alt="pretty picture" />
+          </p>
+        </form>
+      {% endcode %}
+    </figure>
+    </section>
+
+
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-ini">
+        <figcaption>.ini Syntax Highlighting Example</figcaption>
+        {% code 'ini' %}
+-        ; boilerplate
+-        [package]
+-        name = "some_name"
+-        authors = ["Author"]
+-        description = "This is \
+-        a description"
+-
+-        [[lib]]
+-        name = ${NAME}
+-        default = True
+-        auto = no
+-        counter = 1_000
+-      {% endcode %}
+    </figure>
+</section>
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed